clients/android/core/src/main/java/app/lockbook/core/core.kt

```diff
@@ -22,5 +22,6 @@ external fun executeSyncWork(config: String, account: String, workUnit: String):
 
 fun loadLockbookCore() {
     System.loadLibrary("lockbook_core")
-    initLogger()
+    // TODO: @smail put the real config here
```

Comment:
You can just remove this line from both sides and revert changes to `java_interface.rs` we can have @SmailBarkouch deal with this in a different PR